packages/react-aria-components/docs/Modal.mdx

```diff
@@ -101,12 +98,18 @@ import {DialogTrigger, Modal, Dialog, Button, Heading, TextField, Label, Input}
 }
 
 .react-aria-Modal {
+  position: fixed;
+  max-height: var(--visual-viewport-height);
+  top: calc(var(--visual-viewport-height) / 2);
+  left: 50%;
+  translate: -50% -50%;
```

Comment:
Kind of a weird hack to center the modal without adding an extra div wrapper to the example...